flow/project.py

```diff
@@ -3305,29 +3404,25 @@ def select(operation):
                 )
                 break
             try:
-                # Change groups to available run _JobOperation(s)
+                # Generate _JobOperation instances for selected groups and aggregates.
```

Comment:
@atravitz requested an improvement on this comment earlier. 👍

Reply:
much clearer!!